src/rtkpos.c

```diff
@@ -1974,14 +1971,12 @@ static int relpos(rtk_t *rtk, const obsd_t *obs, int nu, int nr,
         rtk->ssat[sat[i]-1].snr_base[j] =obs[ir[i]].SNR[j]; 
     }
     
-    /* initialize Pp,xa to zero, xp to rtk->x */
-    xp=mat(rtk->nx,1); Pp=zeros(rtk->nx,rtk->nx); xa=mat(rtk->nx,1);
-    matcpy(xp,rtk->x,rtk->nx,1);
-    matcpy(Pp,rtk->P,rtk->nx,rtk->nx);
+    /* initialize xa to zero */
```

Comment:
nit Could define x and P here as just rtk->x and rtk->P then use x and P to match the usage in ddres().

Reply:
I wasn't completely sure what exactly you meant, but I added aliases for both `rtk->x` and `rtk->P` now. It's in the last commit. Have a look and let me know whether you meant something different, please!